recipes/ucsc-bedclip/meta.yaml

```
@@ -12,10 +12,13 @@ source:
   sha256: "{{ sha256 }}"
   patches:
     - "include.patch"
+    - "patch1.patch"
 
 build:
   skip: True  # [osx]
-  number: 2
+  number: 3
+  run_exports:
+    - {{ pin_subpackage("ucsc-bedclip", max_pin="x.x") }}
```

```suggestion
    - {{ pin_subpackage("ucsc-bedclip", max_pin=None) }}
```
`ucsc-*` tools do not follow SemVer, otherwise LGTM!